lib/cdo/shared_constants/pd/teacher2021_application_constants.rb

```diff
@@ -8,16 +8,16 @@ def self.clean_multiline(string)
     end
 
     SECTION_HEADERS = {
-      section_1_about_you: 'About You',
-      section_2_choose_your_program: 'Choose Your Program',
-      section_3_teaching_background: 'Teaching Background',
-      section_4_professional_learning_program_requirements: 'Professional Learning Program Requirements',
-      section_5_additional_demographic_information: 'Additional Demographic Information and submission',
+      about_you: 'About You',
+      teaching_background: 'Teaching Background',
+      choose_your_program: 'Choose Your Program',
+      professional_learning_program_requirements: 'Professional Learning Program Requirements',
+      additional_demographic_information: 'Additional Demographic Information and submission',
```

Comment:
This feels good.. the ordering feels less hardcoded now, and more flexible to change order if we discover that we should!